imports/ui/Statistics.jsx

```diff
@@ -1,6 +1,14 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withTracker } from 'meteor/react-meteor-data';
+import { Reviews } from '../api/dbDefs.js';
+import { Classes } from '../api/dbDefs.js';
+import { Students } from '../api/dbDefs.js';
 import Accordian from './Accordian.jsx';
 
+import { LineChart, PieChart } from 'react-chartkick';
+import 'chart.js';
```

Comment:
Where is the place you are using this import?

Reply:
Its used to create the line chart in line 97.

Reply:
Isn't it already handled by
```js
import { LineChart, PieChart } from 'react-chartkick';
```

Reply:
I thought so as well but the chart kick website (https://chartkick.com/react) in their quick start section had both listed. If I remove the chart.js import then chart does not load.